lib/bloc/fee_options/fee_option.dart

```diff
@@ -19,7 +19,9 @@ class FeeOption {
     required this.feeVByte,
   });
 
-  bool isAffordable(int walletBalance) => (walletBalance - fee) > 0;
+  bool isAffordable(int walletBalance) {
+    return (walletBalance - fee) > 0;
+  }
```

Comment:
Keep it as an arrow function.

Reply:
whoops, forgot to change back after trying an alternative solution 

fixed